dvc/stage/run.py

```diff
@@ -93,7 +98,13 @@ def restored_from_cache(stage):
         return False
     # restore stage from build cache
     stage_cache.restore(stage)
-    return stage.outs_cached()
+    restored = stage.outs_cached()
+    if restored:
+        logger.info(
+            "Restored stage %s from run-cache, skipping…",
```

Comment:
```suggestion
            "Restored stage '%s' from run-cache, skipping…",
```
Is run-cache (hyphenated) the official term we will be using consistently?